perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/MembersManagerMethod.java

```diff
@@ -760,12 +760,19 @@ public Void call(ApiCaller ac, Deserializer parms) throws PerunException {
 				}
 			} else {
 				if(parms.contains("allowedStatuses")) {
-					return ac.getMembersManager().findCompleteRichMembers(ac.getSession(),
-							ac.getGroupById(parms.readInt("group")),
-							parms.readList("attrsNames", String.class),
-							parms.readList("allowedStatuses", String.class),
-							parms.readString("searchString"),
-							parms.readBoolean("lookingInParentGroup"));
+					if(parms.contains("group")) {
```

Comment:
Sorry I forgot, please add javadoc for new version of this method in RPC compliant format. Thanks. Then you can merge if you want.